src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -61,6 +62,11 @@
     private static final String AUTH_HEAD = "Basic";
 
     /**
+     * Split pattern.
+     */
+    private static final Pattern PATTERN = Pattern.compile(PsBasic.AUTH_HEAD);
```

Comment:
@krzyk The LineLengthCheck rule allows only 80 characters. Take a look at the entire line 67.